test/no-hot/no-hot.test.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+const { run } = require('../utils/test-utils');
+const { stat } = require('fs');
+const { resolve } = require('path');
+describe('no-hot flag', () => {
```

Comment:
Could you also create a test where we have a config file with `hot: true` and you provide `--no-hot`?

Reply:
Also, a test with just `--no-hot`. It should be successful as test.